www/docs/en/dev/guide/platforms/android/index.md

```diff
@@ -32,276 +32,152 @@ development. For a comparison of the two development paths, see the
 
 ## Requirements and Support
 
+### Java Development Kit (JDK)
+
+Install [Java Development Kit (JDK) 7](http://www.oracle.com/technetwork/java/javase/downloads/jdk7-downloads-1880260.html)
+or later.
+
+When installing on Windows you also need to set `JAVA_HOME` Environment Variable
+according to JDK installation path (for example, `C:\Program Files\Java\jdk1.7.0_75`).
+
+### Android SDK
+
 Cordova for Android requires the Android SDK which could be installed
 on OS X, Linux or Windows operation system. See the Android SDK's
 [System Requirements](http://developer.android.com/sdk/index.html#Requirements).
-
-Cordova supports Android 4.0.x (starting with Android API level 14)
-and higher.  As a general rule, Android versions become unsupported by Cordova as
+Cordova's latest Android package supports up to Android [API-Level](http://developer.android.com/guide/topics/manifest/uses-sdk-element.html#ApiLevels) 23.
+The supported Android API-Levels for the past few cordova-android releases can
+be found in this table:
+
+cordova-android Version | Supported Android API-Levels
+------------------------|-----------------------------
+5.X.X                   | 14 - 23
```

Comment:
I don't have better ideas but this will certainly require some maintenance every time we change the API level.